assets/src/dashboard/app/views/templates/index.js

```diff
@@ -121,18 +143,28 @@ function TemplatesGallery() {
             )}
           />
           <StoryGridView
-            filteredStories={filteredTemplates}
+            filteredStories={orderedTemplates}
             centerActionLabel={__('View details', 'web-stories')}
             bottomActionLabel={__('Use template', 'web-stories')}
```

Comment:
Nit, it looks like `StoryGridView` now has a required prop (`updateStory`):
![image](https://user-images.githubusercontent.com/40646372/80416368-de04ae80-8888-11ea-83a2-3de8272e36fb.png)

Reply:
thanks! updated